src/sentry/testutils/helpers/on_demand.py

```diff
@@ -0,0 +1,55 @@
+from collections.abc import Sequence
+
+from sentry.models.dashboard import Dashboard
+from sentry.models.dashboard_widget import (
+    DashboardWidget,
+    DashboardWidgetDisplayTypes,
+    DashboardWidgetQuery,
+    DashboardWidgetTypes,
+)
+from sentry.models.project import Project
+
+
+def create_widget(
```

Comment:
This is a modified version of the function in here:
https://github.com/getsentry/sentry/blob/f5854efbe04733ef099a3044af431a6aca948e1e/tests/sentry/tasks/test_on_demand_metrics.py#L53-L81